projects/court-case-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/person/entity/Person.kt

```diff
@@ -176,7 +172,15 @@ class Person(
     val restrictionMessage: String? = null,
 
     @Column(columnDefinition = "number")
-    val softDeleted: Boolean = false
+    val softDeleted: Boolean = false,
+
+    @OneToMany(mappedBy = "person")
+    @SQLRestriction("active_flag = 1 and soft_deleted != 1")
```

Comment:
Should this use `<>`?
```suggestion
    @SQLRestriction("active_flag = 1 and soft_deleted <> 1")
```

Or `= 0` may be simpler?
```suggestion
    @SQLRestriction("active_flag = 1 and soft_deleted = 0")
```